src/utils.py

```diff
@@ -88,13 +88,25 @@ def read_masks(masks_filepaths):
             mask_dir = mask_dir[0]
         for i, mask_filepath in enumerate(glob.glob('{}/*'.format(mask_dir))):
             blob = np.asarray(Image.open(mask_filepath))
-            blob_binarized = (blob > 128.).astype(np.uint8) * i
+            blob_binarized = (blob > 128.).astype(np.uint8) * (i + 1)
```

Comment:
Previous version ignored first nucleus, because for the first nucleus `i=0`, so it was dropped.

Reply:
@apyskir so it was buggy before?

Reply:
@jakubczakon Yes, I think so.

Reply:
@apyskir props on spotting it